### DISCUSSION TITLE
Is distribution.cer needed to register my app in Apple Store.

### DISCUSSION THREAD

**Discussion by kanisuki** [airsdk/Adobe-Runtime-Support#3324] [Q&A]
I cannot remember that  distribution.cer is needed or not to regist my app in Apple Store.
I made ios_development.cer and when I publish, I input p12 and mobileprovision.
If I remember correctly  without ditribution.cer I could play Test Fright.

**Comment by marchbold** [comment]
You need a distribution profile and certificate to publish to the app store (including test flight).

**Reply by kanisuki** [reply]
Thank you.
I only to do that about the distribution profile and certificate of my app.
